src/core/version.ts

```diff
@@ -6,13 +6,13 @@ export const CHANNEL_OPTIONS_AGENT_STRING = `chat-js/${VERSION}`;
 export const CHANNEL_OPTIONS_AGENT_STRING_REACT = `chat-react/${VERSION}`;
 // Modes required for basic message functionality
 export const DEFAULT_CHANNEL_MODES: Ably.ChannelMode[] = ['PUBLISH', 'SUBSCRIBE'];
-export const DEFAULT_CHANNEL_OPTIONS: Ably.ChannelOptions = {
+export const DEFAULT_CHANNEL_OPTIONS = {
```

Comment:
I don't see the advantage of this change, isn't it better to explicitly type this as a `ChannelOptions` object if that's how it's used?

Reply:
Fair, reverted back in [a9de20b](https://github.com/ably/ably-chat-js/pull/571/commits/a9de20b65a6791fec370becd522b2c0088b0baa4)